examples/LightStepTestUI/Pods/Target Support Files/lightstep/lightstep-dummy.m

```diff
@@ -0,0 +1,5 @@
+#import <Foundation/Foundation.h>
+@interface PodsDummy_lightstep : NSObject
+@end
+@implementation PodsDummy_lightstep
+@end
```

Comment:
It's something that `pod` creates -- I'll leave it in given my lack of understanding.